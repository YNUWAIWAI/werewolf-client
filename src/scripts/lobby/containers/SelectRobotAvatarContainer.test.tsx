import * as React from 'react'
import {IntlProviderContainer} from './IntlProviderContainer'
import {Provider} from 'react-redux'
import {SelectRobotAvatarContainer} from './SelectRobotAvatarContainer'
import fakeStore from './fakeStore'
import {mount} from 'enzyme'

test('<SelectRobotAvatarContainer />', () => {
  const store = fakeStore()
  const wrapper = mount(
    <Provider store={store} >
      <IntlProviderContainer>
        <SelectRobotAvatarContainer />
      </IntlProviderContainer>
    </Provider>
  )

  expect(wrapper.html()).toMatchSnapshot()
})
