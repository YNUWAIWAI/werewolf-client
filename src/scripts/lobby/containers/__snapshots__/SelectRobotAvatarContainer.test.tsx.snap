// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SelectRobotAvatarContainer /> 1`] = `"<div class=\\"lo--grid\\"><div class=\\"lo--header\\">Play as a robot player</div><div class=\\"lo--main-content\\"><div class=\\"lo--select-avatar--header\\">Select an avatarâ€¦</div><div class=\\"lo--select-avatar--table\\"><div class=\\"lo--select-avatar--table--header--item check-box\\"></div><div class=\\"lo--select-avatar--table--header--item avatar\\">Avatar</div><div class=\\"lo--select-avatar--table--header--item language\\">Language</div><div class=\\"lo--select-avatar--table--header--item status\\">Status</div><div class=\\"lo--select-avatar--table--header--item authorized\\">Authorized</div><div class=\\"lo--select-avatar--table--header--item test\\">Test</div><div class=\\"lo--select-avatar--table--header--item automation\\">Automation</div><div class=\\"lo--select-avatar--table--header--item access-token\\">Access token</div></div><ul class=\\"lo--select-avatar--command\\"></ul></div><div class=\\"lo--aside-content \\"><ul class=\\"lo--compact-menu\\"><li class=\\"lo--compact-menu--item\\"><button class=\\"lo--compact-menu--item--button \\" tabindex=\\"-1\\">Create a new avatar</button></li><li class=\\"lo--compact-menu--item\\"><button class=\\"lo--compact-menu--item--button \\" tabindex=\\"-1\\">Return to the main page</button></li></ul></div></div>"`;
